src/app/search/get-histogram-width.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'getHistogramWidth',
+})
+export class GetHistogramWidthPipe implements PipeTransform {
+  /**
+   * This pipe sums number of items in a facet and returns the width of the histogram as a proportion
+   *
+   * @param {Map<string, number>} bucket The items in one facet
+   * @param {string} subBucket The sub-bucket value (e.g. http://edamontology.org/data_9090)
+   * @returns {number} The width of the histogram for that row
+   * @memberof GetHistogramWidthPipe
+   */
+  transform(bucket: Map<string, number>, subBucket: string): number {
```

Comment:
This was already merged, but pretty sure the type should be Map<string, string>.  That's probably why you need the `Number` on line 24.

https://github.com/dockstore/dockstore-ui2/blob/b59e590eeea9da1c8ca38fdc3bc50d4a0b400d0a/src/app/shared/models/SubBucket.ts#L19